typescript/sdk/src/utils/gnosisSafe.js

```diff
@@ -13,22 +17,61 @@ export function getSafeService(chain, multiProvider) {
   return new SafeApiKit.default({ txServiceUrl, ethAdapter });
 }
 
+// Default safe version to use if not specified
+const DEFAULT_SAFE_VERSION = '1.3.0';
+const safeVersionOverrides = {
+  ancient8: '1.1.1',
```

Comment:
there's no way to get this via the SDK I presume? as this is used for starting to interact with it?

Reply:
there was not in the protocol-kit but after adding the types I found a way in the api-kit which seems to be working! see [`ef00a34` (#4415)](https://github.com/hyperlane-xyz/hyperlane-monorepo/pull/4415/commits/ef00a34559814d2e6ae332815d3fe958069aa367)